deploy/Dockerfile

```diff
@@ -68,7 +68,7 @@ ENV PATH="$PATH:/root/.cargo/bin"
 COPY --from=builder /app/target/release/pipeline-manager pipeline-manager
 # SQL compiler uber jar
 RUN mkdir -p database-stream-processor/sql-to-dbsp-compiler/SQL-compiler/target
-COPY --from=javabuild /sql/sql-to-dbsp-compiler/SQL-compiler/target/sql2dbsp-jar-with-dependencies.jar database-stream-processor/sql-to-dbsp-compiler/SQL-compiler/target/sql2dbsp-jar-with-dependencies.jar
+COPY --from=javabuild /sql/sql-to-dbsp-compiler/SQL-compiler/sql-compiler/target/sql2dbsp-jar-with-dependencies.jar database-stream-processor/sql-to-dbsp-compiler/SQL-compiler/sql-compiler/target/sql2dbsp-jar-with-dependencies.jar
```

Comment:
the hierarchy is already deep, we should probably put them in SQL-compiler and slt both in sql-to-dbsp-compiler. I hope I can adapt this scheme.

Reply:
@mihaibudiu agreed. Remove the capitalized `SQL-compiler/` folder and moving the rest a level up should do the trick.